src/http.js

```diff
@@ -72,13 +71,15 @@ class Http {
 	 * @param {Object} obj
 	 * @returns {Request}
 	 */
-	makeRequest(obj) {
+	async makeRequest(obj) {
```

Comment:
Does this (the function now being async) create any issues?

Reply:
Nope, I fixed all the refrences